DataFormats/CTPPSAlignment/src/RPAlignmentCorrectionsData.cc

```diff
@@ -58,15 +60,18 @@ RPAlignmentCorrectionData RPAlignmentCorrectionsData::GetFullSensorCorrection(un
   bool useRPErrors) const
 {
   RPAlignmentCorrectionData c;
-  mapType::const_iterator it = rps.find(id / 10);
-  if (it != rps.end())
-    c = it->second;
-  it = sensors.find(id);
-  if (it != sensors.end())
-    c.add(it->second, useRPErrors);
 
-  //printf("> full correction %u | ", id);
-  //c.print();
+  // try to get alignment correction of the full RP
+  auto rpIt = rps.find(CTPPSDetId(id).getRPId());
```

Comment:
This line contains the only real modifications with respect to the previous version of the DataFormat  part of the code.
Do "rps.find(id / 10)" and "rps.find(CTPPSDetId(id).getRPId())" return the same ?
If not: what is this line supposed to change?